test/units/executor/module_common/test_recursive_finder.py

```diff
@@ -19,23 +19,18 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
-import imp
 import pytest
 import zipfile
 
 from collections import namedtuple
-from functools import partial
-from io import BytesIO, StringIO
+from io import BytesIO
```

Comment:
This is unused now

Reply:
`BytesIO` is still used in the `finder_containers` fixture.